src/reactions/aggregateReactions.js

```diff
@@ -1,19 +1,24 @@
 /* @flow strict-local */
-import type { Reaction, AggregatedReaction } from '../types';
+import type { Reaction, AggregatedReaction, User } from '../types';
+import { NULL_USER } from '../nullObjects';
 
 export default (
   reactions: $ReadOnlyArray<Reaction>,
   ownEmail: string,
+  users?: User[],
 ): $ReadOnlyArray<AggregatedReaction> =>
   Array.from(
     reactions
       .reduce((reactionMap, x) => {
+        const user = users && users.find((u: User) => u.user_id === x.user.user_id);
+        const reactedUser = user || { ...NULL_USER, ...x.user };
```

Comment:
When will we be unable to find a valid `user`?

Reply:
When user is deactivated, I guess :)